Tasks/ExtractFilesV1/_buildConfigs/Node20/package-lock.json

```diff
@@ -1,368 +1,462 @@
 {
   "name": "ExtractFiles",
   "version": "1.0.2",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
Shouldn't we hold to lockfileVersion: 1?

Reply:
it's generated  by 'buildgen' which is not controller by me.